### PR TITLE
Grouping files in /devel by SIG - SIG Architecture

### PR DESCRIPTION
**Pull request by eduartua**
Ref. #3064 and #3097 

Files owned by SIG Architecture have been moved to a new sig folder. Links have been updated too.

/sig contributor-experience
/cc @parispittman @nikhita @bgrant0607 @jdumars 

/area developer-guide
/kind cleanup

### CODE REVIEW

**Comment by eduartua** [comment]
The last commit includes tombstone files and all updated links in k/community.

**Comment by spiffxp** [comment]
/assign @cblecker @idvoretskyi 
(retrying above minus the sig-api-machinery-leads)

File: contributors/design-proposals/auth/apparmor.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ and is supported on several
 # Alpha Design
 
 This section describes the proposed design for
-[alpha-level](../devel/api_changes.md#alpha-beta-and-stable-versions) support, although
+[alpha-level](https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions) support, although
```
Comment (cblecker) [inline contributors/design-proposals/auth/apparmor.md]: Is there a reason this uses the shortener, as opposed to just `/contributors/devel/sig-architecture/api_changes.md`?

Comment (eduartua) [reply contributors/design-proposals/auth/apparmor.md]: No reason. I'll fix this.

Comment (cblecker) [reply contributors/design-proposals/auth/apparmor.md]: There's a couple more of these.. Can you [review](https://github.com/kubernetes/community/pull/3151/files) and correct in the other places too?

Comment (eduartua) [reply contributors/design-proposals/auth/apparmor.md]: I'll review from the root folder again. Thanks.

**Pull request merged**